### DISCUSSION TITLE
How to change favicon icon color?

### DISCUSSION THREAD

**Discussion by alexUnd** [piroor/treestyletab#3378] [Q&A] [answered]
Hi. I am not strong in css, but I have customized everything for myself, using the instructions to TST. but one setting that I would like to implement, I can not do anything. 
I would like to paint another color icons (favicons) of specific service tabs. I have reviewed all the possible instructions, but my code doesn't do what I want it to do. 
I've tried something like this, but "fill" doesn't work.

`tab-item[data-current-uri^="about:"] tab-favicon {
  fill: rgb(212,34,34) !important;
}`

 would appreciate any help

**Comment by irvinm** [comment, marked as answer]
How about this?

```
tab-item[data-current-uri^="about:"] tab-favicon { 
    filter: brightness(0) saturate(100%) invert(18%) sepia(48%) saturate(5054%) hue-rotate(351deg) brightness(103%) contrast(89%); 
}
```
![image](https://github.com/piroor/treestyletab/assets/979729/7c56fe5b-3bef-481c-b3ba-bff9403851d6)

**Reply by alexUnd** [reply]
Thank you! with changing "hue-rotate" parameter in this code i can match any color!
